pos_ticket_without_price/static/src/xml/pos.xml

```diff
@@ -30,4 +30,35 @@
         </t>
     </t>
 
+    <t t-name="XmlReceiptWithoutPrice" t-extend="XmlReceipt">
+        <t t-jquery="t[t-if='receipt.company.logo']" t-operation="before">
+            <h1>TICKET WITHOUT PRICE</h1>  
+        </t>
+        
+        <t t-jquery="t[t-foreach='receipt.orderlines']" t-operation="replace">
+            <t t-foreach='receipt.orderlines' t-as='line'>
+                 <value t-att-value-decimals='pos.dp["Product Unit of Measure"]' value-autoint='on'>
+                    <t t-esc='line.quantity' />
+                </value>
+                x
+                <t t-esc='line.product_name_wrapped[0]' />
+                <br/>
+            </t>
+        </t>
+
+        <t t-jquery="t[t-set='taxincluded']" t-operation="replace" />
+        <t t-jquery="t[t-set='!taxincluded']" t-operation="replace" />
+        <t t-jquery="t[t-set='taxincluded']" t-operation="replace" />
+
+        <t t-jquery="line &gt; right" t-operation="replace" />
+        <t t-jquery="line &gt; left" t-operation="replace" />
+        
+        <t t-jquery="line[class='total']" t-operation="replace" />
+        <t t-jquery="t[t-foreach='paymentlines']" t-operation="replace" />
+        <t t-jquery="line[size='double-height']" t-operation="replace" />
```

Comment:
Missing code
```suggestion
        <t t-jquery="line[size='double-height']" t-operation="replace" />
        <t t-jquery="t[t-if='receipt.total_discount']" t-operation="replace" />
```

Reply:
<t t-jquery="t[t-if='receipt.total_discount']" t-operation="replace" /> This its missing?

Reply:
That's right, maybe you haven't tested with discounts, but if you don't add that last line, some amounts should be incorrectly displayed. Do you agree?
Don't worry about the previous line, I've only shown it in order to use "suggested change" functionality of GitHub, the line doesn't actually change.